### ISSUE TITLE
Rosetta not finding locale pos in conf/locale

### ISSUE THREAD

**Issue by guillaumepiot** [mbi/django-rosetta#79]
By default, Django will store the locale files in `conf/locale`.

But some reasons, Rosetta will not pick up those files, but rather only files in the app directories.

Any ideas why?

**Comment by mbi** [comment]
Are you referring to Django's own translation catalogs, i.e. point number 3 of https://docs.djangoproject.com/en/1.5/topics/i18n/translation/#how-django-discovers-translations ?

**Comment by guillaumepiot** [comment]
Yes, that's the one, it would be very useful if Rosetta could pick up those locale files as well.

**Comment by juanmitaboada** [comment]
Easy solution:
1) touch app/conf/**init**.py
2) Add app.conf to your apps in settings.py
